ponycheck/test/forall.pony

```diff
@@ -17,3 +25,45 @@ class MultipleForAllTest is UnitTest
 
     Ponycheck.for_all[U8](recover Generators.unit[U8](1) end, h)(
       {(u, h) => h.assert_eq[U8](u, 1, u.string() + " == 1") })?
+
+class ForAll2Test is UnitTest
+  fun name(): String => "ponycheck/for_all2"
+
+  fun apply(h: TestHelper) ? =>
+    Ponycheck.for_all2[U8, String](
+      recover Generators.unit[U8](0) end,
+      recover Generators.ascii() end,
+      h)(
+        {(arg1, arg2, h) =>
+          h.assert_false(arg2.contains(String.from_array([as U8: arg1])))
+          })?
```

Comment:
Minor nit: the closing brace for lambda should be at the same indentation level as the opening one